reframed/tsconfig.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "../tsconfig.json",
-  "files": ["index.ts"]
+  "include": ["global.d.ts", "src"]
```

Comment:
FYI: my personal preference and best practice is to use files over include as `files` forces you to provide just the entry point just like when you configure a bundler. includes and globs sometimes end up pulling in unexpected files which can result in misleading type checking errors. let's not change this now, but I'll likely change it later.